userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md

```diff
@@ -97,20 +117,9 @@ To see all flags available on this subcommand, see [the help](/docs/pctl/pctl-ad
 
 ## The `git-repository` flag
 
-:::tip
-While not required, it is recommended to always set the `--git-repository` flag.
-This flag is needed by the majority of artifact types, so it may be easier to always
-set it rather than have to discover whether a profile requires it.
-
-Keep reading for instructions on how to set this option.
-:::
-
-This flag is required for installing profiles with local resources.
-A local resource is any profile artifact which refers to manifests stored within the profile
-repository. You can see [this page](/docs/author-docs/local-helm-chart) from the author docs as an example.
-
-In order to install profiles with local resources, you must provide `pctl` with information
-about your GitOps repo (very meta, I know).
+This flag is required for installing profiles because pctl generate flux `Kustomization` resources for
+deploying the profile artifacts, and these resources need to know what `GitRepository` resource is governing the repository.
```

Comment:
```suggestion
This flag is required for installing profiles because pctl generates flux `Kustomization` resources for
deploying the profile artifacts, and these resources need to know which `GitRepository` resource is governing the repository.
```